.changeset/orange-years-rush.md

```diff
@@ -0,0 +1,5 @@
+---
+"@commercetools/sdk-middleware-http": patch
+---
+
+Allow requests with empty content-type
```

Comment:
Please specify that you need to explicitly pass `null` to "unset" the header.

Reply:
Hi @emmenko 
This is how I worded it: To unset a request header's content-type, explicitly pass `null` as the value. 

Hope it's fine?